src/machine/machine_stm32l4.go

```diff
@@ -117,6 +117,25 @@ const (
 	PE15 = portE + 15
 )
 
+// IRQs are defined here as they vary in the SVDs, but do have consistent mapping
+// to Timer Interrupts.
+const (
+	irq_TIM1_BRK_TIM15     = 24
```

Comment:
Perhaps you want to put these into `device/stm32` similar to how https://github.com/tinygo-org/tinygo/blob/release/src/device/sam/atsamd51x-bitfields.go is?

Reply:
How strongly do you feel about it?

I kinda feel icky about putting hand-crafted code in there.  Long-term better to work with rust team on getting the IRQ constants aligned.  I've made these non-exported so they don't become an implicit API for apps.

Reply:
I like your long-term solution much better!